extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "DavidAnson.vscode-markdownlint"
```

Comment:
This file can be removed but I think it may aid contributors if they install the most popular markdownlint extension

<img width="723" alt="Screen Shot 2022-07-26 at 2 30 41 PM" src="https://user-images.githubusercontent.com/3699047/181085526-3be469ef-a9e5-4057-a4f8-ca14c5706edd.png">
 
so that contributors can catch markdownlint violations at development time rather than at CI time.

Reply:
Can you split this up into a separate PR just so it doesn't block? Some devs don't use vscode so this could result in bikeshedding.

Reply:
Extracted to https://github.com/celestiaorg/celestia-node/issues/949